tests/mir-opt/building/eq_never_type.rs

```diff
@@ -0,0 +1,12 @@
+// skip-filecheck
```

Comment:
I tried to use `// CHECK`, but could make it work, because they seem to be run on the optimized MIR, at which point everything is dead-code-eliminated.

Reply:
You can add a `-Zmir-opt-level=0`, which will avoid opts.

Reply:
The call is deleted even with `-Zmir-opt-level=0` :(